### PR TITLE
fixed aws url link, and added missing hyphen

### PR DESCRIPTION
**Pull request by jcayouette**
# Description

In response to:

https://github.com/uyuni-project/uyuni-docs/issues/830

# Target branches

Which documentation version does this PR apply to?

- [X] Master (Default)
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes https://github.com/uyuni-project/uyuni-docs/issues/830

### CODE REVIEW

**Review by calancha — APPROVED** [review]
LGTM

**Comment by calancha** [comment]
My bad, we don't need to add the dash.  What is said there is that SUMA reports the ID w/o dash; that is correct, as you
can see here:
-id-seen-from-suma-Screenshot from 2021-03-04 14-28-28

**Comment by Loquacity** [comment]
> My bad, we don't need to add the dash. What is said there is that SUMA reports the ID of the instance as an UUID w/o dash; that is correct, as you can see here:
> -id-seen-from-suma-Screenshot from 2021-03-04 14-28-28

Looks like it needs to be a capital letter "I", not lowercase?

File: modules/client-configuration/pages/vhm-aws.adoc [hunk 1]
```diff
@@ -38,5 +38,5 @@ To see which objects and resources have been inventoried, navigate to menu:Syste
 Instances running on the Amazon public cloud report a UUID to the {productname} Server in the format of an ``i`` followed by seventeen hexadecimal digits:
 
 ----
-i1234567890abcdef0
+i-1234567890abcdef0
```
Comment (Loquacity) [inline modules/client-configuration/pages/vhm-aws.adoc:41]: Looks like this might need to be an uppercase "I", not lowercase?

Comment (calancha) [reply modules/client-configuration/pages/vhm-aws.adoc:41]: That's right: in SUMA, it is shown in uppercase.

Comment (jcayouette) [reply modules/client-configuration/pages/vhm-aws.adoc]: Fixed

**Pull request merged**